plugins/woocommerce-blocks/package.json

```diff
@@ -48,10 +48,8 @@
 		"changelog": "XDEBUG_MODE=off composer install --quiet && composer exec -- changelogger",
 		"build": "pnpm --if-present --workspace-concurrency=Infinity --stream --filter=\"$npm_package_name...\" '/^build:project:.*$/'",
 		"build:project": "pnpm --if-present '/^build:project:.*$/'",
-		"build:project:validate-blocks-manifests": "wireit",
 		"build:project:bundle": "wireit",
-		"build:project:blocks-json-to-php": "wireit",
-		"build:check-assets": "rimraf build/* && cross-env ASSET_CHECK=true BABEL_ENV=default NODE_ENV=production webpack",
+		"build:check-assets": "npx ajv validate -s bin/block.json-validation-schema.json -d assets/js/**/block.json && rimraf build/* && cross-env ASSET_CHECK=true BABEL_ENV=default NODE_ENV=production webpack",
```

Comment:
@samueljseay @mikejolley: sanity check, `build:check-assets` script in blocks seems unused (introduced in #5634). Can you confirm I've mot missed any usages?

Reply:
The `package.json` files for Blocks & Admin still have some crust from iteration in their respective repositories before they were merged. I didn't want to break any workflows and so they were never removed but things like this should be cleaned up.

Reply:
Gotcha, many thanks @ObliviousHarmony! I'll drop this command, then (probably with ASSET_CHECK if no other usages).

Reply:
To be followed up in https://github.com/woocommerce/woocommerce/pull/55325